### DISCUSSION TITLE
Syntax errors in policy.

### DISCUSSION THREAD

**Discussion by aambadkar** [learndevopsonline/learndevopsonline#1516] [Q&A] [answered]
hi,
 i am trying to create roles and their policies, so role are created but getting error for policy
![Capture](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/4bad7233-a35d-4cf9-a65d-5316ef6f7f6c)

![Capture1](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/0e5ef606-4866-4a73-bd8c-8fdce88e4988)

**Comment by sreedharm07** [comment]
Hi @aambadkar 
check line by line that you copied json file correctly or just add policy json file again and i observed that your code is different in your repo and the screenshot code which you attached here. please give git pull too and try once

**Reply by aambadkar** [reply]
hi, 
role and policy are created but at the time of task execution having error Failed to access SSM parameter expense.frontend.backend_url (AccessDenied)"}
![Capture](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/59414123-b4e4-4d50-95ad-5aa2ce8c45b7)

**Reply by sreedharm07** [reply]
can you please check **expense.frontend.backend_url**  in aws parameter store weather parameter is created or not.

**Reply by aambadkar** [reply]
yes its created
 
![Capture](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/a0e043de-eeb9-4a0a-99a0-1cad42587854)

**Reply by sreedharm07** [reply, marked as answer]
because there is no policy is attached for dev-frontend role to get parameters.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/afc6e5bb-d9cd-47f6-8d81-86d5ad2196a9)


![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/2bbbdf9e-3e00-4d37-bc1f-42cf6066a58e)

**Reply by aambadkar** [reply]
but its seems that policy is attached for dev-frontend role
![Capture](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/0a5529d6-a4f8-4044-8573-b56d2037c634)
